### DISCUSSION TITLE
Mestrado para graduação tecnologica?

### DISCUSSION THREAD

**Discussion by KumoMg** [frontendbr/forum#1136] [Discussão]
Bom dia!
Lendo algumas issues antigas eu percebi que existe uma boa quantidade de pessoas aqui que fizeram graduação tecnologica.

Eu estou cursando graduação tecnologica em analise de sistemas e gostaria de tentar continuar meus estudos no exterior após a graduação, por isso eu queria saber se é possivel fazer um mestrado lá fora com um diploma de graduação tecnologica ?(a dúvida é se apenas diplomas de "bacharelado" seriam visto como diplomas validos)

Alguem aqui conseguiu fazer mestrado lá fora? Se sim, poderias compartilhar o nome do pais e talvez o da universidade ?

Abraços!

**Comment by tunnes** [comment]
@KumoMg Necessariamente fora? Em SP existem bons cursos na USP e UFABC, recentemente venho pensando bastante em ingressar em um. A UFABC valoriza bastante os formados tecnólogos.
